### DISCUSSION TITLE
How to get initial driver race positions?

### DISCUSSION THREAD

**Discussion by locvio** [br-g/openf1#114] [General]
What's the best way to determine initial race positions? (starting grid)
Documentation for positions says
"Provides driver positions throughout a session, including initial placement and subsequent changes."
but from the data it looks like it is updates only..

I need a sorted list of positions from beginning, and at a given time. If I had easy way to get initial positions can maintain state using position updates..
thanks..

https://api.openf1.org/v1/position?meeting_key=1217

**Comment by marclelamy** [comment]
look at the earliest datetime?

**Comment by locvio** [comment]
The data only has updates when positions change…earliest date time would get you first position change for one driver, I believe..On Nov 23, 2024, at 11:33 PM, Marc Lamy ***@***.***> wrote:﻿
look at the earliest datetime?
